Core/include/Acts/Seeding/Seedfinder.ipp

```diff
@@ -277,20 +277,127 @@ void Seedfinder<external_spacepoint_t, platform_t>::createSeedsForGroup(
       scatteringInRegion2 *=
           m_config.sigmaScattering * m_config.sigmaScattering;
 
+      float sinTheta = 1 / std::sqrt(iSinTheta2);
+      float cosTheta = cotThetaB * sinTheta;
+
       // clear all vectors used in each inner for loop
       state.topSpVec.clear();
       state.curvatures.clear();
       state.impactParameters.clear();
       for (size_t t = t0; t < numTopSP; t++) {
         auto lt = state.linCircleTop[t];
 
+        float cotThetaT = lt.cotTheta;
+        float cotThetaBNew;
+        float rMxy;
+        float ub;
+        float vb;
+        float ut;
+        float vt;
+
+        if (m_config.useDetailedDoubleMeasurementInfo == true) {
```

Comment:
I think it's somewhat confusing that you sometimes check for 
`m_config.useDetailedDoubleMeasurementInfo == true` and sometimes for `m_config.useDetailedDoubleMeasurementInfo == false`. Is there a reason you're not simply always checking 
`m_config.useDetailedDoubleMeasurementInfo`?

Reply:
Done